src/_includes/styles.css

```diff
@@ -217,36 +286,30 @@ footer.footer {
 
 ul.nav-links {
     list-style: none;
-    padding: 0;
-    margin: 1em;
-}
-
-ul.nav-buttons li {
-    display: inline-block;
-    margin: 2px;
+    padding: 1em 0;
+    margin: 0 auto;
 }
 
-ul.nav-buttons a {
+nav.header a {
     display: inline-block;
-    padding: 5px 10px;
-    margin: 0px 10px;
-    border: 1px solid white;
```

Comment:
Originally the focus style updated just the `border-color` part, but leaned on a `border-width` of `1px` being set here.  If we add the width in, things will jank around, and `outline` doesn't seem to respect the `border-radius` property, so I'm going to add in a `box-shadow` on focus

Reply:
ooh I see.  My bad for missing that.  We could also put the 1px border back with the same color as the background.  That way it isn't visible until you change the color on focus.

Reply:
oh, yeah, or set `1px solid transparent` (although the explicit color probably gives us something to `transition` against).  Already pushed `box-shadow`, but we can update if we'd like